sql/moz-fx-data-shared-prod/fivetran_costs_derived/monthly_costs_v1/query.sql

```diff
@@ -38,8 +38,10 @@ monthly_costs AS (
       monthly_costs_credits.measured_month,
       monthly_costs_dollars.measured_month
     ) AS measured_month,
-    monthly_costs_credits.credits_spent,
-    monthly_costs_dollars.dollars_spent
+    COALESCE(
+      monthly_costs_credits.credits_spent * 2,
+      monthly_costs_dollars.dollars_spent
+    ) AS dollars_spent
```

Comment:
Actually since the dollars and credits are distinct from each other I can just do a union all here. Does feel more elegant? 

And yes the number comes from checking the contracts.

Reply:
Sure, a union would be reasonable since there's no overlap, though in that case I'd guard against Fivetran possibly sending `credits_used` data in the future by adding a date condition (probably on both queries for consistency/clarity).

Or similarly, if you keep the full outer join + coalesce then to be safest I'd put the `dollars_spent` first in the coalesce so it takes precedence over any future `credits_used` data.